ec_tools/helper.py

```diff
@@ -12,7 +12,7 @@ def find_x0_values(x, y, mode="all"):
 
     TODO:
```

Comment:
Maybe you should open an issue for the Todo and add the issue number
```suggestion
    TODO:: see #X
```

Reply:
```suggestion
    TODO:: see #5
```